src/main/webapp/dev/js/common/bootboxWrapper.es6

```diff
@@ -28,12 +28,16 @@ function showModalAlert(titleText, messageText, okButtonText, color) {
         buttons: {
             okay: {
                 label: okButtonText || DEFAULT_OK_TEXT,
-                className: `modal-btn-ok btn-${color}` || StatusType.DEFAULT,
+                className: `modal-btn-ok btn-${color || StatusType.DEFAULT}`,
             },
         },
     })
-    // applies bootstrap color to title background
-    .find('.modal-header').addClass(`alert-${color}` || StatusType.DEFAULT);
+    .find('.modal-header')
```

Comment:
Yup close! I think `bootbox.dialog` is a function though, so it probably wouldn't work? It should look more like:
```javascript
const modal = bootbox.dialog({
...
});
applyStyleForModal(modal, statusType);
```